README.md

```diff
@@ -70,33 +222,13 @@ For a more informal place to chat, if you worry about feeling dumb in the open o
 ## <a name="team"></a> Team
 
 Core contributors :
```

Comment:
Maybe we could have an old core contributors' list ?
In order to not forget about the old warriors :||